components/region_cache_memory_engine/src/keys.rs

```diff
@@ -235,6 +235,19 @@ pub fn encode_key_for_eviction(range: &CacheRange) -> (InternalBytes, InternalBy
     (encoded_start, encoded_end)
 }
 
+#[inline]
+pub fn encode_key_for_boundary_without_mvcc(range: &CacheRange) -> (InternalBytes, InternalBytes) {
+    // Both encoded_start and encoded_end should be the smallest key in the
+    // respective of user key (without mvcc version), so that the iterations covers
```

Comment:
> "should be the smallest key in the respective of the user key".

could you clarify what you mean by this? smallest possible valid key prefix? smallest possible user key in that range?

Reply:
Keys in kv engine are format of  user_key + mvcc_version + sequence number.
So the smallest key in the respective of user key is user_key + max mvcc version + max sequence number